.github/ISSUE_TEMPLATE/NEW_PROPOSAL.md

```diff
@@ -0,0 +1,50 @@
+
+---
+name: "\U0001F4D6 New Proposal"
+about: Template for creating a new proposal
+title:
+labels: 
+assignees: ''
+
+---
+<!--
+Please complete the following sections when you open a new proposal issue.
+-->
+
+## Title of the session: WRITE HERE
+<!-- Please provide a short title of your proposal capturing what your target audience would expect in this session.
+-->
+
+### Session details
+
+* Session type: WRITE HERE 
+<!-- Choose from these options: Breakout discussion, skill-up, social event, lightening talk, informal meetup, panel, pub quiz on specific topic, or other -->
+
+* Keywords: WRITE HERE 
+<!--Please provide 3-5 keywords to help your expected target audience to decide the relevance of this session-->
+
+* Permission to record this session: WRITE HERE 
+<!-- Please asnwer in 'Yes' or 'No', Since this session will be possibly taking place in a time zone that is not compatible for everyone, we would liek to record the presentation/introduction and conclusion/final wrap up part of your session through Zoom recording option.-->
+
+### Abstract
+<!-- Please provide a short  -->
```

Comment:
do we want to set some limit on the abstract size? for example 300 words?

Reply:
I can't remember if we had a word limit for the first round of proposals. I don't have strong feelings for/against.

Reply:
yes, 250 words. looks like I totally zoned out there.